<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZappyUI</class>
 <widget class="QWidget" name="ZappyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="viewer">
      <attribute name="title">
       <string>2.5DView</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Q2DZappy" name="sfmlView" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="preview">
      <attribute name="title">
       <string>Moniteur</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Nombre de zappies vivants</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Nom des équipes</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Nombre d'oeufs</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Nombre de zappies morts</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLCDNumber" name="nb_ded"/>
          </item>
          <item row="10" column="3">
           <widget class="QTextEdit" name="zappiesBroadcast">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Messages des zappies :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLCDNumber" name="nb_alive"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Nombre de zappies</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3">
           <widget class="QLineEdit" name="teamNames">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLCDNumber" name="nb_zappies"/>
          </item>
          <item row="5" column="3">
           <widget class="QLCDNumber" name="nb_cycles_saw"/>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nombre de cycles surveillés</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QLCDNumber" name="nb_resources"/>
          </item>
          <item row="1" column="3">
           <widget class="QLCDNumber" name="height_map"/>
          </item>
          <item row="0" column="3">
           <widget class="QLCDNumber" name="length_map"/>
          </item>
          <item row="7" column="3">
           <widget class="QLCDNumber" name="nb_eggs"/>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Nombre ressources</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Hauteur de la map</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Largeur de la map</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="zappies">
      <attribute name="title">
       <string>Zappies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableau">
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Zappie</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Position</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Niveau</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Oeufs pondus</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nouriture</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Linemate</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Deraumère</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sibur</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mendiane</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Phiras</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Thystame</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q2DZappy</class>
   <extends>QWidget</extends>
   <header>q2dzappy.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
